components/widgets/menu/menu.jsx

```diff
@@ -30,13 +30,13 @@ export default class Menu extends React.PureComponent {
     }
 
     render() {
-        const {children, openUp, openLeft, onMobile, id, ariaLabel} = this.props;
+        const {children, openUp, openLeft, mobile, id, ariaLabel} = this.props;
         const styles = {};
-        if (openLeft && (!isMobile() || !onMobile)) {
+        if (openLeft && (!isMobile() || mobile === false)) {
```

Comment:
Since this is fixing the correct props, you may retain the same implementation as
```suggestion
        if (openLeft && (!isMobile() || !mobile)) {
```

Same for instance below.

Reply:
@jespino Any thoughts on this?

Reply:
@deanwhillier Any thoughts on this?

Reply:
@saturninoabril, the `mobile` prop is not a required prop so can be undefined causing the conditional to be true on mobile thereby adding the non-mobile inline styles that are only meant for non-mobile views. This is what the original problem was.

Maybe having this additional prop is getting too confusing? If it's really an override of `isMobile()` for use when we want to force mobile behaviour of the menu, we should change the prop name to something more appropriate? Something like `forceMobile` or `renderMobile`? Alternatively we can just remove the prop until it's actually needed and leave the decision till later?

Reply:
Or perhaps re-format the conditional like so:

``` javascript
if (openLeft && !(isMobile || mobile)) {
```

Then it doesn't matter if the `mobile` prop is undefined.

Reply:
I just removed it for now. We can add it later.